internal/fixture/state.go

```diff
@@ -504,3 +508,46 @@ func (tc TestCase) TestWithConverter(parser Parser, converter merge.Converter) e
 
 	return nil
 }
+
+// PrintState is an Operation printing the current state to help with debugging tests
+type PrintState struct{}
```

Comment:
Yes, multiple tests would solve that as well.
While creating (and debugging) the remove tests, it was already getting less comfortable working with them due to the amount of tests. Also because of my unfortunate decision not to name them...

For now I only wanted to understand and debug the broken tests that already exist.
Pulling them apart into multiple tests would have been something for another PR and then still might cause more clutter while the Print and Expect Operations would allow for the same result in the existing test fixture thing.